src/components/title/_fields/contributor/contributor-field.js

```diff
@@ -34,12 +34,17 @@ class ContributorField extends Component {
             autoFocus={Object.keys(fields.get(index)).length === 0}
             label={intl.formatMessage({ id: 'ui-eholdings.type' })}
```

Comment:
I think we can clean up these labels as well

Reply:
Happening in a separate PR 😁